### PR TITLE
O3-1678 Order basket medication search results should have two-button layout

### PR DESCRIPTION
**Pull request by brandones**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [x] My work includes tests or is validated by existing tests.

## Summary
Implements the [designs](https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/62c6bb9f143fb81962e39594) for medication search results.

## Screenshots
image

## Related Issue
https://issues.openmrs.org/browse/O3-1678

## Other
Unrelated: fixed the lab order panel extension order. It had been the same as the one for meds, meaning the order was non-deterministic.

### CODE REVIEW

**Comment by denniskigen** [comment]
@brandones based on the interaction in the [Add and remove items from basket](https://zeroheight.com/23a080e38/p/71ad91-order-basket) section of the design doc, should clicking on `Add to basket` navigate the user away from the search page and back to the order basket landing page showing that there's an updated drug order available.

**Comment by brandones** [comment]
Thanks Dennis, good catch. I had been looking for that ZeroHeight site and hadn't been able to find it. Should we add it to the [OpenMRS 3.0 wiki page](https://wiki.openmrs.org/display/projects/OpenMRS+3.0%3A+A+Frontend+Framework+that+enables+collaboration+and+better+User+Experience) alongside the link to Zeplin?

I ended up completely reworking the CSS for the workspace. I think it's in much better shape.

Desktop:
https://github.com/openmrs/openmrs-esm-patient-chart/assets/1031876/3176bfce-2a83-4aaf-9a29-d764c7e12363

Tablet:
https://github.com/openmrs/openmrs-esm-patient-chart/assets/1031876/0994496b-3515-46bc-98c9-fee0b81c9196

**Comment by denniskigen** [comment]
Absolutely we should, @brandones.

**Comment by brandones** [comment]
Just for posterity, here is what the workspace looked like before this change. Demoing a few visual bugs. Some are introduced by https://github.com/openmrs/openmrs-esm-patient-chart/pull/1378; those are not fixed in this PR.

https://github.com/openmrs/openmrs-esm-patient-chart/assets/1031876/06014f25-00b7-4d36-b3a1-83963d22e8fb

File: packages/esm-patient-medications-app/translations/es.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (brandones) [reply packages/esm-patient-medications-app/translations/es.json]: Yeesh, translation management is awkward. Okay should be fixed.

**Review by denniskigen — APPROVED** [review]
Excellent work, @brandones. Thanks.

File: packages/esm-patient-medications-app/translations/es.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (denniskigen) [reply packages/esm-patient-medications-app/translations/es.json]: Yeah, the current approach is not great.

**Comment by brandones** [comment]
Yeah @denniskigen sorry, I meant to mention this earlier: that's sort of a complex problem/feature so I'm planning to address it as a separate ticket. https://issues.openmrs.org/browse/O3-2427

**Pull request merged**